app/src/main/java/edu/uc/zhukv/droneradarmap/MapsActivityCurrentPlace.kt

```diff
@@ -43,17 +44,17 @@ class MapsActivityCurrentPlace(var NameOfCity: TextView? = null) {
                     == PackageManager.PERMISSION_GRANTED) {
                 var locationPermissionGranted = true
             } else {
-                val PERMISSIONS_REQUEST_ACCESS_FINE_LOCATION = 0
+
                 //ActivityCompat.requestPermissions(this, arrayOf(Manifest.permission.ACCESS_FINE_LOCATION),
                         //PERMISSIONS_REQUEST_ACCESS_FINE_LOCATION)
             }
 
 
-    fun onRequestPermissionsResult(requestCode: Int,
+    internal fun onRequestPermissionsResult(requestCode: Int,
                                             permissions: Array<String>,
                                             grantResults: IntArray) {
         var locationPermissionGranted = false
-        val PERMISSIONS_REQUEST_ACCESS_FINE_LOCATION = null
+
```

Comment:
Removed null assignment to variable of non-null type, (new constant declaration at top), as recommended from: https://kotlinlang.org/docs/null-safety.html

Reply:
Very good.